plasma_framework/contracts/src/exits/utils/BondSize.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.5.0;
+
+import 'openzeppelin-solidity/contracts/math/SafeMath.sol';
+
+library BondSize {
+    using SafeMath for uint256;
+
+    struct Params {
+        uint256 currentGasPrice;
+        uint256 challengeGasCost;
+        uint256 ewmaFactor;
+        uint256 safetyFactor;
+    }
+
+    function bondSize(Params storage _self) internal view returns (uint256) {
```

Comment:
I think this can be `function bondSize(Params memory _self)`. With `memory`, the function can take both `storage` and `memory` struct as input and would be more flexible.